### PR TITLE
add netcoreapp1.0 target

### PR DESCRIPTION
**Pull request by analogrelay**
right now it duplicates the build but that's OK.

/cc @pakrym @piotrpMSFT @Sridhar-MS

### CODE REVIEW

File: test/test.xunit.runner.dnx/project.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ﻿{
   "frameworks": {
-    "netstandardapp1.5": {
+    "netcoreapp1.0": {
```
Comment (eerhardt) [inline test/test.xunit.runner.dnx/project.json:3]: Do we still want to test both TFMs?

Comment (TheRealPiotrP) [inline test/test.xunit.runner.dnx/project.json:3]: Can we do both here as well? Would be good to know we're not regressing netstandardapp...

**Comment by eerhardt** [comment]
One small question.

:shipit:  This is going to be great

**Comment by analogrelay** [comment]
I'm going to leave it unmerged right now, @piotrpMSFT can you and @Sridhar-MS merge it either when the CI is up and running or when you're ready to make a manual build with the change? I don't have access to push a manual build to MyGet...

It is ready to merge now though, feedback addressed

File: test/test.xunit.runner.dnx/project.json [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
       "imports": [
         "dnxcore50",
         "portable-net45+win8"
       ]
     }
   },
-	"testRunner":"xunit",
+  "testRunner":"xunit",
   "dependencies": {
     "Microsoft.NETCore.App": "1.0.0-rc2-23931",
```
Comment (TheRealPiotrP) [inline test/test.xunit.runner.dnx/project.json:18]: @anurse doesn't this need to be type=platform for netcoreapp?

**Comment by TheRealPiotrP** [comment]
Merging the PR. Let's talk about making it portable.

**Pull request merged**